src/components/Assessment.js

```diff
@@ -59,11 +59,11 @@ const Assessment = (props) => {
   }, [])
 
   return (
-    <div className="px-12 sm:px-32">
+    <div className="px-12 md:px-32">
       {assess.map((assessment, index) => {        
         return (
           <div id="week" key={index} className="w-full border-b border-black py-10">
-            <div className="flex w-full flex-row justify-between">
+            <div id="dashboard-table" className="flex w-full flex-row justify-between">
```

Comment:
What is dashboard table? Is there something more descriptive this can be named?

Reply:
Removed...